ts/src/test/static/credentials/woofipro.json

```diff
@@ -0,0 +1,3 @@
+{
+    "secret": "secretsecretsecretsecretsecretsecretsecrets"
```

Comment:
we don't need this, we can read it from the the test file

Reply:
what do you mean?

Reply:
I will update it don't worry